keras_cv/layers/preprocessing/random_choice.py

```diff
@@ -52,19 +52,26 @@ class RandomChoice(BaseImageAugmentationLayer):
             apply the augmentations. This offers a significant performance
             boost, but can only be used if all the layers provided to the
             `layers` argument support auto vectorization.
+        batchwise: (Optional) bool, whether to pass entire batches to the
```

Comment:
Let's mention that the default is `False`.

IMO the docstring would be a bit easier to follow as something like:
"Whether to chose a layer on a batchwise basis instead of an example-wise basis." wdyt?

Reply:
Improved the docstring a bit!